libs/SmartContracts.js

```diff
@@ -410,7 +410,6 @@ class SmartContracts {
       }
```

Comment:
were there changes that didn't go through?

Reply:
Not sure what do you mean here. All the actions were moved back to the tokens contract

Reply:
nevermind, it's just that the changeset seems to be empty, just wanted to make sure you didn't forget to commit something